Backend-llm/tests/test_streaming.py

```
@@ -0,0 +1,253 @@
+async def test_streaming_with_empty_message():
+    """Test streaming with empty message handling."""
+    messages = [{"role": "user", "content": ""}]
+    
+    chunks = []
+    try:
+        async for chunk in stream_chat_with_model("openai", "gpt-4o", messages):
+            chunks.append(chunk)
+        
+        # Should either complete with a response or return an error
+        assert len(chunks) > 0, "Should receive at least one chunk"
+        assert chunks[-1]["done"] == True, "Should complete"
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `True`; use `chunks[-1]["done"]:` for truth checks

Replace with `chunks[-1]["done"]`

(E712)